amplitude_python_sdk/common/utils/__init__.py

```diff
@@ -0,0 +1,18 @@
+"""This package contains utility functions used in both V1 and V2 SDK methods."""
```

Comment:
I think this file should be deleted now, since `utils` is just a file in the `common` package instead of a separate package.